City-of-Helsinki/drupal-module-helfi-tpr
# UHF-1031: Opening hours

To test this:

- Run `drush updb`
- Migrate some TPR unit entities (`MIGRATE_LIMIT=500 drush mim tpr_unit`)
- Migrate TPR unit extras with `drush mim tpr_unit_extra`

Enable Opening hours field display (Structure -> Tpr unit -> Manage display)

Make sure you can see opening hours. 

Run `drush sqlq "SELECT distinct(entity_id) FROM tpr_unit__opening_hours;"` to list unit ids with opening hours.
